ocs_ci/ocs/constants.py

```diff
@@ -1244,3 +1248,20 @@
 VOLUMESNAPSHOT = "volumesnapshot"
 
 PERF_IMAGE = "quay.io/ocsci/perf:latest"
+
+ROOK_CEPH_CONFIG_VALUES = [
+    "[global]",
+    "mon_osd_full_ratio = .85",
+    "mon_osd_backfillfull_ratio = .8",
+    "mon_osd_nearfull_ratio = .75",
+    "mon_max_pg_per_osd = 600",
+    "[osd]",
+    "osd_memory_target_cgroup_limit_ratio = 0.5",
+]
```

Comment:
This should be stored as a single string.

Reply:
Is there a specific reason? 
I added it as a list so it will be easier to iterate over while validating the different items

Reply:
I would rather still store it in a single piece, and split it into list of lines in the test, since this is placed in constants module, expected to be used by other code (using this value requires to assume that this is a list of strings, and since list is mutable, it's not a constant and test code can't 100% rely that it's value hasn't been changed by mistake).

Reply:
Got you, thanks for explaning.
Fixed now